src/Workspaces/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions_Parsing.cs

```diff
@@ -49,5 +50,47 @@ private static string GetExpressionBodyPreferenceEditorConfigString(CodeStyleOpt
                     throw new NotSupportedException();
             }
         }
+
+        public static CodeStyleOption<UsingPlacementPreference> ParseUsingPlacementPreference(
+            string optionString, CodeStyleOption<UsingPlacementPreference> @default)
+        {
+            // optionString must be similar to outside_namespace:error or inside_namespace:suggestion.
+            if (CodeStyleHelpers.TryGetCodeStyleValueAndOptionalNotification(
+                optionString, out var value, out var notificationOpt))
+            {
+                // A notification value must be provided.
+                if (notificationOpt != null)
+                {
+                    switch (value)
+                    {
+                        case "no_preference":
```

Comment:
❓ Why not `preserve`? The current name sounds like using directives are not allowed (which is actually a code style I've seen before).